xml/depl_multiple_networks.xml

```diff
@@ -0,0 +1,187 @@
+          }
+    },</screen>
+</step>
+
+<step>
+    <para>
+         Install the modified &o_netw; &barcl; and modify 
+<parameter>additional_external_networks</parameter> with the name of 
+        your new external network.
+    </para>        
+</step>
+</procedure>
+</sect1>
+
+<sect1 xml:id="sec.confignet">
+    <title>Configure the New External Network</title>
+        <para>
+        The following steps add the network settings, including IP address 
+pools, gateway, routing, and virtual switches to your new network. 
+</para>
+
+    <procedure xml:id="pro.confignet">
+<step>
+    <para>
+Set up interface mapping using either Open vSwitch (OVS) or Linuxbridge. 
+For Open vSwitch run the following command:
+</para>
+<screen>sudo neutron net-create <replaceable>public2</replaceable>: --provider:network_type flat \
+ --provider:physical_network <replaceable>public2</replaceable>: --router:external=True</screen>
+ 
+ <para>
+For Linuxbridge run the following command:
+</para>
+
+<screen> sudo neutron net-create --router:external True --provider:physical_network physnet1 \
+ --provider:network_type vlan --provider:segmentation_id <replaceable>600</replaceable></screen>
+</step>
+
+<step>
+    <para>
+        If a different network is used then &crow; will create a new interface 
+mapping. Then you can use a flat network:
+</para>
+<screen>
+sudo neutron net-create <replaceable>public2</replaceable> --provider:network_type flat \
+ --provider:physical_network <replaceable>public2</replaceable> --router:external=True
+</screen>
+</step>
+
+<step>
+    <para>
+Create a subnet:     
+</para>
+<screen>sudo neutron subnet-create --name <replaceable>public2</replaceable> --allocation-pool \
+ start=<replaceable>192.168.135.2</replaceable>,end=<replaceable>192.168.135.127</replaceable> --gateway <replaceable>192.168.135.1</replaceable> <replaceable>public2</replaceable> \
+ <replaceable>192.168.135.0/24</replaceable> --enable_dhcp False</screen>
+</step>
+
+<step>
+    <para>
+        Create a router, <replaceable>router2</replaceable>:  
+</para>
+<screen>sudo neutron router-create <replaceable>router2</replaceable></screen>
+</step>
+
+<step>
+    <para>
+ Connect <replaceable>router2</replaceable> to the new external network:       
+</para>
+<screen>sudo neutron router-gateway-set <replaceable>router2</replaceable>  <replaceable>public2</replaceable></screen>
+</step>
+
+<step>
+    <para>
+        Create a new private network and connect it to 
+<replaceable>router2</replaceable>
+</para>
+<screen>sudo neutron net-create priv-net
+sudo neutron subnet-create priv-net --gateway <replaceable>10.10.10.1 10.10.10.0/24</replaceable> \
+ --name priv-net-sub
+sudo neutron router-interface-add <replaceable>router2</replaceable> priv-net-sub</screen>
+</step>
+
+<step>
+    <para>
+ Boot a VM setting the web security group and the <literal>priv-net-sub</literal> 
+network.       
+</para>
```

Comment:
@rossella Not sure what this means, setting the web security group and priv-net-sub to what?